backend/src/email/routes/tests/email-transactional.routes.test.ts

```diff
@@ -863,6 +865,161 @@ describe(`${emailTransactionalRoute}/send`, () => {
     ])
   })
 
+  const ccValidTests = [
+    ['cc-recipient@agency.gov.sg', 'cc-recipient-2@agency.gov.sg'],
+    JSON.stringify([
+      'cc-recipient@agency.gov.sg',
+      'cc-recipient-2@agency.gov.sg',
+    ]),
```

Comment:
Are we able to have a test case with just a single cc recipient?

Reply:
good catch, added test for single recipient